integration/simulation/simulation_tester.go

```diff
@@ -45,5 +49,5 @@ func testSimulation(t *testing.T, netw network.Network, params *params.SimParams
 
 	// generate and print the final stats
 	t.Logf("Simulation results:%+v", NewOutputStats(&simulation))
-	netw.TearDown()
+	log.Info("goroutine leak monitor - simulation end - %d go routines currently running", runtime.NumGoroutine())
```

Comment:
@BedrockSquirrel Isn't it misleading to log the remaining goroutines before stopping the network? What if we did the following?

```
defer go() {
    netw.TearDown()
	log.Info("goroutine leak monitor - simulation end - %d go routines currently running", runtime.NumGoroutine())
}()
```

Reply:
Yeah good spot. This might help get the numbers down haha.